phpfarm/docker.sh

```diff
@@ -17,14 +17,23 @@ do
 
     # compile the PHP version
     ./compile.sh $VERSION
+
+    # PHP 5.1 names both CLI and CGI executables 'php'. Rename and move them to
+    # correct dirs. See http://php.net/manual/bg/features.commandline.php
+    if [ "$V" == "5.1" ]; then
+        post='custom/post-install-5.1.6.sh'
+        echo "Running commands from '$post'"
+        /bin/bash "$post" "$VERSION"
+    fi
```

Comment:
I added 2 patches in _custom/patches-5.1.6/_ so the compile and install works properly now:
* [cgi.patch](https://github.com/splitbrain/docker-phpfarm/pull/13/files#diff-bb93ff79708c2d9213cd46e8990d3344) for _sapi/cgi/config9.m4_. This fixes the CGI compiled binary filename to _php-cgi_.
* [configure.patch](https://github.com/splitbrain/docker-phpfarm/pull/13/files#diff-86b859bb183e0a4d2beab6456b06959f) for _configure_. This fixes the CGI compiled binary filename, and also makes the CLI compilation happen. Before this, only the CGI was compiled on _make install_. To compile the CLI version, you had to do _make install-cli_.

I've also removed the post-install steps from _docker.sh_, as _compile.sh_ from _fpoirotte/phpfarm_ is now able to detect and run _custom/post-install-5.1.6.sh_.